public/js/index.js

```diff
@@ -1,5 +1,32 @@
-function displayMessage() {
```

Comment:
I appreciate your recommendations but to be honest I don't have experience with karma so may you please fix it in a separate pull request?